src/sentry/consumers/__init__.py

```diff
@@ -396,12 +415,43 @@ def build_consumer_config(group_id: str):
             healthcheck_file_path, strategy_factory
         )
 
+    if enable_dlq:
+        try:
+            dlq_topic = consumer_definition["dlq_topic"]
+        except KeyError as e:
```

Comment:
Is this still needed? Haven't we already validated?

Reply:
Still needed, because they're loaded dynamically using django lazy settings